src/frameworks/drupal8/solr.md

```diff
@@ -14,17 +53,38 @@ And then commit the changes to `composer.json` and `composer.lock`.
 
 ## Configuration
 
-Go to `/admin/config/search/search_api` and click on *Add Server*. Give
-the server a name and a description.
+The configuration can be managed from `settings.platformsh.php` by adding this code snippet:
 
--   Server class: *Solr service*
--   Solr host: `solr.internal`
--   Solr port: `8080`
--   Solr path: `/solr`
+```php
+<?php
+// Configures the Solr server.
+if (isset($_ENV['PLATFORM_RELATIONSHIPS'])) {
+  $relationships = json_decode(base64_decode($_ENV['PLATFORM_RELATIONSHIPS']), TRUE);
 
-> **note**
-> The name of the host depends on the name you gave the relationship in `.platform.app.yaml`. If
-> you called the relationship `solrsearch` instead, then the hostname will be `solrsearch.internal`.
+  // The Solr relationship name defined in your '.platform.app.yaml' file. In
+  // this example it is 'solr1'.
+  $relationship_name = 'solr1';
+
+  if (!empty($relationships[$relationship_name][0])) {
+    $solr = $relationships[$relationship_name][0];
+
+    // The the machine name of the Solr server in Drupal. The default server
+    // automatically created by Search API is 'default_solr_server'.
+    $solr_server_name = 'default_solr_server';
+
+    // The path is always 'solr'.
+    $config['search_api.server.' . $solr_server_name]['backend_config']['connector_config']['path'] = 'solr';
+
+    // The name of the Solr core defined in your '.platform/services.yaml'
+    // file. In this example it is 'maincore'
+    $config['search_api.server.' . $solr_server_name]['backend_config']['connector_config']['core'] = 'maincore';
```

Comment:
Let's extract this from the relationship path value.  It should be simple enough to do.